ceph/ceph_admin/helper.py

```diff
@@ -391,6 +393,111 @@ def generate_rgw_spec(self, spec):
 
         return template.render(spec=spec)
 
+    def exec_command(self, node, **kwargs: Any) -> Tuple:
```

Comment:
can we make use of read().decode() at called function instead of having separate definition.